lib/krikri/engine.rb

```diff
@@ -56,5 +56,32 @@ class Engine < ::Rails::Engine
         .register(:oai, Krikri::Harvesters::OAIHarvester)
     end
 
+    initializer :rdf_repository do
+      Krikri::Repository =
+        RDF::Marmotta.new(Krikri::Settings['marmotta']['base'])
+    end
+
+    initializer :aggregation do
+      DPLA::MAP::Aggregation.class_eval do
+        include Krikri::LDP::RdfSource
+        configure :base_uri => Krikri::Settings['marmotta']['item_container']
```

Comment:
Curious about what precipitated this. Don't have a strong feeling about it though.